test/services/cap_api_search_results_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'application_system_test_case'
+
+class CapApiSearchResultsTest < ApplicationSystemTestCase
```

Comment:
Seeing as this is a service test, does it need all the system test machinery? Could we make a `ServiceTestCase < TestCase` class that just pulls in the CapApi/ActionMailer stubs and whatever else these actually depend on to run?